### PR TITLE
Background coloring fixes for dialog / pages

### PR DESCRIPTION
**Pull request by rainemak**
File picking needs still a fix

### CODE REVIEW

File: src/pages/components/Theme.qml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+pragma Singleton
+import QtQml 2.2
+
+QtObject {
+    property color backgroundColor: "black"
+    property real backgroundOpacity: 0.7
```
Comment (rainemak) [inline src/pages/components/Theme.qml]: Having a singleton Theme for Browser is a bit over kill as Background.qml can handle both of these.

What do you think?

File: src/pages/components/UserPrompt.qml [hunk 2]
```diff
@@ -11,6 +11,7 @@
 
 import QtQuick 2.0
 import Sailfish.Silica 1.0
+import "." as Browser
```
Comment (rojkov) [inline src/pages/components/UserPrompt.qml]: unused import?

Comment (rainemak) [inline src/pages/components/UserPrompt.qml]: True that... Will wipe out.

**Comment by rainemak** [comment]
Fixed file pickers, review comments, and removed unnecessary browser Theme singleton.

**Comment by rojkov** [comment]
LGTM

**Comment by tworaz** [comment]
LGTM

**Pull request merged**